e2e/whisper/whisper_mailbox_test.go

```diff
@@ -81,7 +82,7 @@ func (s *WhisperMailboxSuite) TestRequestMessageFromMailboxAsync() {
 		],
 		"id": 1
 	}`)
-	msgFilterResp := newMessagesFilterResponse{}
+	msgFilterResp := returnedIDResponse{}
```

Comment:
what does "returnedIDResponse" mean? "return" and "response" both in variable name seems a bit generic to me

Reply:
Maybe `responseWithID`?